precious.toml

```diff
@@ -0,0 +1,28 @@
+# We could run this via golangci-lint but we want to exclude more directories
+# for gosec than we do for other linting. There doesn't seem to be a way to
+# configure these excludes when running gosec as part of golangci-lint.
+[commands.gosec]
+type = "lint"
+include = "**/*.go"
+exclude = [
+    "buildscript/**/*.go",
+    "common/testutil/**/*.go",
+    "release/**/*.go",
+    "vendor/**/*.go",
+]
+invoke.per-dir-or-once = 7
+working-dir = "root"
+path-args = "dir"
+# This wrapper lets us control the output with an env var so we can generate a
+# SARIF report in Evergreen but use the default format for linting.
+cmd = [
+    "$PRECIOUS_ROOT/etc/gosec-wrapper.sh",
```

Comment:
Is it required to run the script on Windows as well? It seems the rest of the release script all support Windows.

Reply:
This will not need to run on Windows. We only need this to run in CI on _one_ distro, not all of them.